hs_core/hydroshare/resource.py

```diff
@@ -305,25 +309,26 @@ def create_resource(
     :param metadata: list of dicts containing keys (element names) and corresponding values as
     dicts { 'creator': {'name':'John Smith'}}.
     :param extra_metadata: one dict containing keys and corresponding values
-    { 'Outlet Point Latitude': '40', 'Outlet Point Longitude': '-110'}.
+         { 'Outlet Point Latitude': '40', 'Outlet Point Longitude': '-110'}.
     :param files: list of Django File or UploadedFile objects to be attached to the resource
-    :param fed_res_file_names: the file names separated by comma from a federated zone to be
-                               used to create the resource in the federated zone, default is empty
-                               string
+    :param source_names: an array of file names separated by comma from a federated zone to be
+         used to create the resource in the federated zone, default is empty array
```

Comment:
@alvacouch The docstring for source_names does not sound right to me. If it is a list, there is no need for ```file names separated by comma``` but instead all file names are stored as a list, right?